src/overlays/actors/ovl_Bg_Dy_Yoseizo/z_bg_dy_yoseizo.c

```diff
@@ -277,30 +273,30 @@ void BgDyYoseizo_ChooseType(BgDyYoseizo* this, GlobalContext* globalCtx) {
             if (globalCtx->sceneNum != SCENE_DAIYOUSEI_IZUMI) {
                 switch (this->fountainType) {
                     case FAIRY_SPELL_FARORES_WIND:
-                        globalCtx->csCtx.segment = SEGMENTED_TO_VIRTUAL(&D_02000160);
+                        globalCtx->csCtx.segment = SEGMENTED_TO_VIRTUAL(&gGreatFairyFaroresWindCs);
```

Comment:
Is the `&` needed here? Since cutscenes are just arrays, it shouldn't be needed.
```suggestion
                        globalCtx->csCtx.segment = SEGMENTED_TO_VIRTUAL(gGreatFairyFaroresWindCs);
```